server/protocols/webadmin-integration-test/distributed-webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/rabbitmq/RabbitMQEventDeadLettersIntegrationTest.java

```diff
@@ -207,8 +232,8 @@ private String retrieveFirstFailedInsertionId() {
     }
 
     @Test
-    public void failedEventShouldBeStoredInDeadLetterUnderItsGroupId() {
-        retryEventsListener.callsPerEventBeforeSuccess(MAX_RETRIES + 1);
+    void failedEventShouldBeStoredInDeadLetterUnderItsGroupId() {
+        EVENT_EXTENSION.retryEventsListener().callsPerEventBeforeSuccess(MAX_RETRIES + 1);
```

Comment:
This, looks like evil to me.

A more idiomatic way to do this (that also comply to visibility best practices) is to use `@Register` on eventExtension, then make it resolve parameter of type `RetryEventsListener`, and finally pass a parameter to the test classes?